ArchiSteamFarm/Bot.cs

```diff
@@ -1173,8 +1173,17 @@ public sealed class Bot : IDisposable {
 			// We must escape our message prior to sending it
 			message = Escape(message);
 
-			for (int i = 0; i < message.Length; i += maxMessageLength) {
-				string messagePart = message.Substring(i, Math.Min(maxMessageLength, message.Length - i));
+			int i = 0;
+			while (i < message.Length) {
+				int partLength;
+				if (message.Length - i > maxMessageLength) {
+					int lastNewLine = message.LastIndexOf(Environment.NewLine, i + maxMessageLength, maxMessageLength);
```

Comment:
When doing `LastIndexOf` of string, don't forget `StringComparison.Ordinal`.

Reply:
Does it even influence newlines?

Reply:
Yes, performance will be improved since the runtime will use binary sorting order as opposed to culture-specific.